src/IB/ConstraintIBMethod.cpp

```diff
@@ -635,7 +636,7 @@ ConstraintIBMethod::initializeHierarchyOperatorsandData()
     const bool from_restart = RestartManager::getManager()->isFromRestart();
     if (!from_restart) calculateVolumeElement();
     setInitialLagrangianVelocity();
-
+    
```

Comment:
👎 Are you clang-formatting? :-D

Reply:
I am, but maybe this got committed earlier without clang formatting. Let me force clang-format on this file.